src/apis/print.js

```diff
@@ -1,6 +1,10 @@
 /**
  * Print the current document.
  * @method WebViewerInstance#print
+ * @param {boolean} [allPages] If true, all pages will be printed
+ * @param {Array.<number>} [pagesToPrint] If allPages is false and pagesToPrint is passed in, will print only the pages passed in
+ * @param {boolean} [includeAnnotations] If true, will print the documents with the annotations
+ * @param {boolean} [includeComments] If true, will append comments to the document printed
```

Comment:
I think it'd be good to document what the options will be. One way you can do this is
```
{boolean} [allPages=false]
```
or just have something like `Default is false` at the end of line